go/client/cmd_pgp_gen.go

```diff
@@ -68,6 +73,32 @@ func (v *CmdPGPGen) Run() (err error) {
 	return err
 }
 
+// Duplicates the test in libkb.PGPGenArg.CreatePGPIDs() for interactive input
+var CheckPGPID = libkb.Checker{
+	F: func(s string) bool {
+		if !strings.Contains(s, "<") && libkb.CheckEmail.F(s) {
+			return true
+		}
+		_, err := libkb.ParseIdentity(s)
```

Comment:
This check accepts input like `a` and `.` and `@@ <> `.   I think you have to distinguish between the end-of-input case and the error case, otherwise some erroring user input will be silently ignore (which will surprise users)